main.cpp

```diff
@@ -52,106 +52,105 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: car
+//  action 1: the car accelerates
+car.accelerate();
+//  action 2: the car turns on the check engine light
+car.checkEngineLightOn();
+//  action 3: the car detects low windshield wiper fluid
+car.lowWindshieldWiperFluid();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: person
+//  action 1: the person walks
+person.walk();
+//  action 2: the person jumps
+person.jump();
+//  action 3: the person speaks
+person.speak();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: stereo
+//  action 1: the stereo selects an input
+stereo.selectInput();
+//  action 2: the stereo changes the output volume
+stereo.changeOutputVolume();
+//  action 3: the stereo engages the subwoofer
+stereo.engageSubwoofer();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.